tileview/src/main/java/com/qozix/tileview/tiles/Tile.java

```diff
@@ -254,8 +262,13 @@ void reset() {
     }
     mState = State.UNASSIGNED;
     mRenderTimeStamp = null;
-    if( mBitmap != null && !mBitmap.isRecycled() ) {
-      mBitmap.recycle();
+    if( mBitmap != null ) {
+      BitmapRecycler recycler = mBitmapRecyclerReference.get();
```

Comment:
In the case of the `execute( TileRenderPoolExecutor )` call, `mBitmapRecyclerReference` is null, so we get a `NullPointerException` here.